### ISSUE TITLE
client.players_season_totals not Returning Expected Values

### ISSUE THREAD

**Issue by sumeshsharma1** [jaebradley/basketball_reference_web_scraper#275]
Hi there, thanks for a great package. Been tremendous help to me over the years.

Upgraded to the newest version and noticed that `client.players_season_totals` always returns an empty list, no matter the year that is put in. I suspect this may be due to underlying changes on the BBRef side. 

```
>>> client.players_season_totals(2018)
[]
```

Tried debugging this locally by changing the underlying `rows_query` [here](https://github.com/jaebradley/basketball_reference_web_scraper/blob/v4/basketball_reference_web_scraper/html.py#L283) to the following:

```
return """
            //table[@id="totals_stats"]
            /tbody
            /tr[
                not(contains(@class, "thead")) 
                and not(contains(@class, "over_header"))
                and not(contains(@class, "rowSum"))
            ]
        """
```
which returned data from the table but is not returning player information such as slug, name, and team in the output.

**Comment by Deltron2020** [comment]
Just went to work on a project that has been using this package for a while and noticed the same thing...... -_- I thought I was going crazy, did you find a resolution or is this package now dead in the water?

**Comment by sumeshsharma1** [comment]
I was able to get it working locally for my use case by changing the following in `html.py`:

- Changing the `rows_query` [here ](https://github.com/jaebradley/basketball_reference_web_scraper/blob/v4/basketball_reference_web_scraper/html.py#L283) to 
```
return """
            //table[@id='totals_stats']/tbody/tr[
                not(contains(@class, 'thead')) and 
                not(.//td[@data-stat='name_display' and text()='League Average'])
            ]
        """
```
- Updated the `PlayerSeasonTotalsRow` class [here](https://github.com/jaebradley/basketball_reference_web_scraper/blob/v4/basketball_reference_web_scraper/html.py#L541) to
```
class PlayerSeasonTotalsRow(PlayerBoxScoreRow, PlayerIdentificationRow):
    def __init__(self, html):
        super().__init__(html=html)

    @property
    def name(self):
        cells = self.html.xpath('./td[@data-stat="name_display"]')
        if cells:
            link = cells[0].xpath('./a')
            if link:
                return link[0].text_content().strip()
            return cells[0].text_content().strip()
        return ''

    @property
    def slug(self):
        cells = self.html.xpath('./td[@data-stat="name_display"]')
        if cells:
            link = cells[0].xpath('./a')
            if link:
                href = link[0].get('href')
                return href.split('/')[-1].split('.')[0]
        return None

    @property
    def position_abbreviations(self):
        return self._get_cell_value('pos')

    @property
    def age(self):
        return self._get_cell_value('age')

    @property
    def games_played(self):
        return self._get_cell_value('games')

    @property
    def games_started(self):
        return self._get_cell_value('games_started')

    @property
    def team_abbreviation(self):
        return self._get_cell_value('team_name_abbr')

    @property
    def is_combined_totals(self):
        return self.team_abbreviation == "TOT"

    def _get_cell_value(self, data_stat):
        cells = self.html.xpath(f'./td[@data-stat="{data_stat}"]')
        if cells:
            return cells[0].text_content().strip()
        return ''
```

Did not do any larger scale testing, but the above got my code to work properly.

**Comment by AliFakharany** [comment]
should we have a pull request based on the above? having the same issue

**Comment by jaebradley** [comment]
Sorry for the delay @sumeshsharma1 

This should be resolved as of [`v4.14.1`](https://github.com/jaebradley/basketball_reference_web_scraper/releases/tag/4.14.1).

Here's [the `repl.it`](https://replit.com/@jaebradley/PlayerSeasonTotals#pyproject.toml) I used to generate the screenshotted output below.

<img width="1716" alt="image" src="https://github.com/user-attachments/assets/144f13f0-d188-4a29-8077-85ed0c1be5dc">

<img width="1725" alt="image" src="https://github.com/user-attachments/assets/b98b485a-febf-411f-9a89-2342e75537b7">
